FlowCrypt/Functionality/Services/Compose Message Service/ComposeMessageService.swift

```diff
@@ -114,6 +117,23 @@ final class ComposeMessageService {
         let replyToMimeMsg = input.replyToMime
             .flatMap { String(data: $0, encoding: .utf8) }
 
+        if let password = contextToSend.messagePassword, password.isNotEmpty {
+            if subject.lowercased().contains(password.lowercased()) {
+                throw MessageValidationError.subjectContainsPassword
+            }
+
+            let storedPassphrases = passPhraseService.getPassPhrases().map(\.value)
+            if storedPassphrases.contains(password) {
+                throw MessageValidationError.notUniquePassword
+            }
```

Comment:
Ere these stored pass phrases only, or a combination of stored pass phrases + whatever is currently available in memory for in-memory pass phrases?

Reply:
`getPassPhrases` returns `encryptedStorage.getPassPhrases() + inMemoryStorage.getPassPhrases()` (https://github.com/FlowCrypt/flowcrypt-ios/blob/master/FlowCrypt/Functionality/Services/Local%20Private%20Key%20Services/PassPhraseService.swift#L113), so we check against all current user passphrases

Reply:
In that case the variable you test should be renamed to `allAvailablePassPhrases`